server/notification-providers/discord.js

```diff
@@ -47,6 +57,13 @@ class Discord extends NotificationProvider {
 
             // If heartbeatJSON is not null, we go into the normal alerting loop.
             if (heartbeatJSON["status"] === DOWN) {
+                const discordDisplayName = notification.discordUsername || "Uptime Kuma";
+                let threadQuery = "";
+
```

Comment:
don't think that declaring this twice is nessesary (?)

```suggestion
                let threadQuery = "";
```